### DISCUSSION TITLE
Two tables with a sync column [Aug 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6158] [Discord Threads]
> 🙋‍♂️ **.tucado**  
## Hello, is there a way to build a second table with the first column that is dynamically filled with the same values as the first one?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1123925793131741305/image.png)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Creating a view is the easy option. You can hide/unhide fields to reduce cluttering.    
####    
#### Other option, if you insist on having two tables with a common column content - is to create 'after insert' webhook in first column & on its trigger, insert data into second table.  
#  
> 🙋‍♂️ **.tucado**  
#### Thank you. Is there any guide how to do that 'after insert'?    
#### Thank you.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1123925793366609990)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
